sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/python.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans.logical
```

Comment:
I'm quite confusing about where to put the python plans.

For the scala ones, we have a `object.scala` under `org.apache.spark.sql.catalyst.plans.logical` package which contains all encoder related logical plans, and we have a `objects.scala` under `org.apache.spark.sql.execution` package which contains all encoder related physical plans.

For the python ones, current we put all of them under `org.apache.spark.sql.execution.python` package, including logical plans, physical plans, rules, etc.

cc @rxin